doc/api/util.md

```diff
@@ -126,6 +126,10 @@ util.format(1, 2, 3); // '1 2 3'
 
 ## util.inherits(constructor, superConstructor)
 
+_Note: usage of `util.inherits()` is discouraged. Please use ES6 `class` and
+`extends` to get language level inheritance support, including an unbroken
+inheritance chain._
```

Comment:
The problem is not the call to `Object.setPrototypeOf()` itself. There are two missing things in `util.inherits()` that justify using `extends`:

- The constructor has to be written by the user (as showed by @bnoordhuis)
- The `Base` class should be in the prototype chain of `Desc` (to give access to static methods). Instead we have `Desc.super_`